Code/Framework/AzCore/Tests/Math/QuaternionTests.cpp

```diff
@@ -510,4 +510,16 @@ namespace UnitTest
     }
 
     INSTANTIATE_TEST_CASE_P(MATH_Quaternion, QuaternionScaledAxisAngleConversionFixture, ::testing::ValuesIn(RotationRepresentationConversionTestQuats));
+
+    TEST(MATH_Quaternion, ShortestEquivalent)
+    {
+        const AZ::Quaternion testQuat = AZ::Quaternion::CreateRotationX(AZ::Constants::HalfPi * 3.0f);
+
+        AZ::Quaternion absQuat = testQuat;
+        absQuat.ShortestEquivalent();
+        EXPECT_TRUE(absQuat.IsClose(testQuat.GetShortestEquivalent(), 1e-6f));
+
+        const float angle = absQuat.GetEulerRadians().GetX();
+        EXPECT_TRUE(AZ::IsClose(-AZ::Constants::HalfPi, angle, 1e-4f));
```

Comment:
I would use the `FloatEq` matcher here, because it will give better output when the test fails:
```suggestion
        EXPECT_THAT(angle, testing::FloatEq(-AZ::Constants::HalfPi));
```

Reply:
Changed